### ISSUE TITLE
BinaryFocalCrossentropy loss not working

### ISSUE THREAD

**Issue by CORRELU** [tensorflow/tensorflow#58588]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Bug

### Source

source

### Tensorflow Version

2.9.1

### Custom Code

No

### OS Platform and Distribution

Windows 10

### Python version

3.9

### CUDA/cuDNN version

11.2

### GPU model and memory

NVIDIA Quadro T2000

### Current Behaviour?

```shell
I'm not able to declare BinaryFocalCrossentropy loss function when apply_class_balancing or alpha arguments are provided.
```

### Standalone code to reproduce the issue

```shell
import tensorflow as tf
loss = tf.keras.losses.BinaryFocalCrossentropy(
    apply_class_balancing=False,
    alpha=0.25,
    gamma=2.0,
    from_logits=False,
    label_smoothing=0.0
)
```

### Relevant log output

```shell
Traceback (most recent call last):
  File "C:\Users\User\AppData\Roaming\Python\Python39\site-packages\IPython\core\interactiveshell.py", line 3251, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-8-235cf9a4168d>", line 1, in <module>
    loss = tf.keras.losses.BinaryFocalCrossentropy(
TypeError: __init__() got an unexpected keyword argument 'apply_class_balancing'
```
</details>

**Comment by tilakrayal** [comment]
@CORRELU,
Thanks for opening this issue. Development of keras moved to another [repository](https://github.com/keras-team/keras/issues). 
 
 
Could you please post this issue on keras-team/keras [repo](https://github.com/keras-team/keras/issues).
To know more please refer:
https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999

**Comment by gowthamkpr** [comment]
@CORRELU Thank you for reporting the bug. This has been fixed in tensorflow >= 2.10 . Please take a look at the [gist](https://colab.sandbox.google.com/gist/gowthamkpr/8e95ebc774766db70b04d1303cc76daa/untitled405.ipynb)!!
